src/UsgsAstroLsSensorModel.cpp

```diff
@@ -1713,19 +1713,30 @@ void UsgsAstroLsSensorModel::losToEcf(
    // Remove lens distortion
    double isisFocalPlaneX = isisNatFocalPlaneX;
    double isisFocalPlaneY = isisNatFocalPlaneY;
-  if (m_opticalDistCoef[0] != 0.0 ||
-     m_opticalDistCoef[1] != 0.0 ||
-     m_opticalDistCoef[2] != 0.0)
+   switch (m_ikCode)
    {
-      double rr = isisNatFocalPlaneX * isisNatFocalPlaneX
-         + isisNatFocalPlaneY * isisNatFocalPlaneY;
-      if (rr > 1.0E-6)
+   case -85610:
+   case -85600:
+      isisFocalPlaneY = isisNatFocalPlaneY / (1.0 + m_opticalDistCoef[0]
+         * isisNatFocalPlaneY * isisNatFocalPlaneY);
+      break;
+
+   default:
+      if (m_opticalDistCoef[0] != 0.0 ||
+         m_opticalDistCoef[1] != 0.0 ||
+         m_opticalDistCoef[2] != 0.0)
       {
-         double dr = m_opticalDistCoef[0] + (rr * (m_opticalDistCoef[1]
-            + rr * m_opticalDistCoef[2]));
-         isisFocalPlaneX = isisNatFocalPlaneX * (1.0 - dr);
-         isisFocalPlaneY = isisNatFocalPlaneY * (1.0 - dr);
+         double rr = isisNatFocalPlaneX * isisNatFocalPlaneX
+            + isisNatFocalPlaneY * isisNatFocalPlaneY;
+         if (rr > 1.0E-6)
+         {
+            double dr = m_opticalDistCoef[0] + (rr * (m_opticalDistCoef[1]
+               + rr * m_opticalDistCoef[2]));
+            isisFocalPlaneX = isisNatFocalPlaneX * (1.0 - dr);
+            isisFocalPlaneY = isisNatFocalPlaneY * (1.0 - dr);
+         }
       }
+      break;

```

Comment:
Is there a reason this is being added back in?

Reply:
Nope, weird git addition with a reset - I'll clean it out